grammars/ruby.cson

```diff
@@ -1940,6 +1952,10 @@
     'name': 'punctuation.separator.key-value'
   }
   {
+    'match': '->'
+    'name': 'keyword.operator.lambda.ruby'
```

Comment:
Yes - my understanding is that in CoffeeScript the arrows stand for the `function` keyword with (`=>`) and without (`->`) binding (w/ JS also having `=>` but no `->`). In Ruby the `->` is an operator that serves as a shortcut to the Kernel#lambda method, so it should be tokenized differently than in CS. That said, there may be a better match than 'keyword.operator.lambda.ruby' if people have other ideas.